doc/guides/env_vars.rst

```diff
@@ -0,0 +1,144 @@
+.. _env_vars:
+
+Environment Variables
+=====================
+
+Various pages in this documentation refer to setting Zephyr-specific
+environment variables. This page describes how.
+
+Setting Variables
+*****************
+
+Option 1: Just Once
+===================
+
+To set the environment variable :envvar:`MY_VARIABLE` to ``foo`` for the
+lifetime of your current terminal window:
+
+.. code-block:: console
+
+   # Linux and macOS
+   export MY_VARIABLE=foo
+
+   # Windows
```

Comment:
Just cmd.exe, not powershell

Reply:
I'm not going to fix this one though, because as stated above, it's a commonplace in our documentation that this means cmd.exe.